api/v2/performanceprofile_types.go

```diff
@@ -73,8 +73,7 @@ type CPU struct {
 	// except the reserved CPUs. In order to guarantee that your workload will run on the isolated CPU:
 	//   1. The union of reserved CPUs and isolated CPUs should include all online CPUs
 	//   2. The isolated CPUs field should be the complementary to reserved CPUs field
-	// +optional
-	Isolated *CPUSet `json:"isolated,omitempty"`
+	Isolated *CPUSet `json:"isolated"`
```

Comment:
/hold
Just until  @cynepco3hahue answers

Reply:
So currently I removed omitempty only from fields that we strictly request to be presented, otherwise, the validation will fail.
See https://github.com/openshift-kni/performance-addon-operators/blob/af31700a496376c6cf70508ba0e65289479bc127/pkg/controller/performanceprofile/components/profile/profile.go#L23.
Probably we can change it, but the change should be done under the different PR.

Reply:
ok